samples/rest/safety_settings.sh

```diff
@@ -0,0 +1,38 @@
+set -eu
+
+echo "[START safety_settings]"
+# [START safety_settings]
+echo '{
+    "safetySettings": [
+        {'category': 7, 'threshold': 3}
```

Comment:
Prefer names instead of integers (the full names work here, right?)

Reply:
Just try it `bash samples/rest/safety_settings.sh`

Reply:
https://source.corp.google.com/piper///depot/google3/google/ai/generativelanguage/v1main/safety.proto

Reply:
done.